docs/internals/security.txt

```diff
@@ -235,3 +241,9 @@ Please also bear in mind that for any individual or organization,
 receiving security notifications is a privilege granted at the sole
 discretion of the Django development team, and that this privilege can
 be revoked at any time, with or without explanation.
+
+.. admonition:: Provide all required information
+
+    A failure to provide the required information in your initial contact
+    will count against you when making the decision on whether or not to
+    approve your request.
```

Comment:
Does it actually count negative? (Maybe it does... 🙂)

```suggestion
    We cannot process you request unless you provide all the required information. 
```

I don't know. 🤷

Reply:
Based upon direct comments from other team members when deciding and my personal views, I'd say it does hurt their chances of approval. It's a bit of a challenge to trust a person with confidential information after they've shown they don't read/follow instructions. 

With that said, I'm not opposed to toning down the wording.

Reply:
OK, let's keep it. Thanks!